chains/container-chains/runtime-templates/frontier/src/xcm_config.rs

```diff
@@ -386,6 +389,7 @@ impl pallet_assets::Config<ForeignAssetsInstance> for Runtime {
     type CallbackHandle = ();
     type AssetAccountDeposit = ForeignAssetsAssetAccountDeposit;
     type RemoveItemsLimit = frame_support::traits::ConstU32<1000>;
+    type Holder = (); // TODO: revisit this
```

Comment:
```suggestion
    type Holder = ();
```

This one is fine, it is set to `()` everywhere

https://github.com/paritytech/polkadot-sdk/pull/4530